src/utils/dataset_db/csv_to_db.py

```diff
@@ -0,0 +1,32 @@
+from db import upload_many
+import csv
+import sys
+
+
+def csv_upload(csv_path):
+    """
+    Given a CSV file, this method uploads it to the organizations-unlabelled
+    collection.
+    """
+    orgs = []
+    with open(csv_path, "r") as csv_file:
+        csv_reader = csv.reader(csv_file, delimiter=",")
+        line_count = 0
+        for row in csv_reader:
+            if line_count == 0:
```

Comment:
what is this for? Could we instead start `line_count` at 1 and explicitly raise an exception if no records are found?

Reply:
Yeah I threw one out, it's mostly to tell you how many records it read.